ui/src/app/utils/runtime-utils.tsx

```diff
@@ -334,10 +375,14 @@ export const useRuntimeStatus = (currentWorkspaceNamespace, currentGoogleProject
     initializePolling();
   }, [runtimeStatus]);
 
-  const setStatusRequest = async(req) => {
+  const setStatusRequest = async(req, deletePD) => {
```

Comment:
I would instead add a new `Request` type to the enum: `DeleteRuntimeAndPD`, then remove the boolean